hack/cherry_pick_pull.sh

```diff
@@ -16,22 +16,24 @@
 
 # Checkout a PR from GitHub. (Yes, this is sitting in a Git tree. How
 # meta.) Assumes you care about pulls from remote "upstream" and
-# checks thems out to a branch named:
+# checks them out to a branch named:
 #  automated-cherry-pick-of-<pr>-<target branch>-<timestamp>
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
-declare -r KUBE_ROOT="$(dirname "${BASH_SOURCE}")/.."
-cd "${KUBE_ROOT}"
+declare -r REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd "${REPO_ROOT}"
 
 declare -r STARTINGBRANCH=$(git symbolic-ref --short HEAD)
-declare -r REBASEMAGIC="${KUBE_ROOT}/.git/rebase-apply"
+declare -r REBASEMAGIC="${REPO_ROOT}/.git/rebase-apply"
 DRY_RUN=${DRY_RUN:-""}
 REGENERATE_DOCS=${REGENERATE_DOCS:-""}
 UPSTREAM_REMOTE=${UPSTREAM_REMOTE:-upstream}
 FORK_REMOTE=${FORK_REMOTE:-origin}
+MAIN_REPO_ORG=${MAIN_REPO_ORG:-$(git remote get-url $UPSTREAM_REMOTE | awk -F'[@:./]' 'NR==1{print $4}')}
+MAIN_REPO_NAME=${MAIN_REPO_NAME:-$(git remote get-url $UPSTREAM_REMOTE | awk -F'[@:./]' 'NR==1{print $5}')}
```

Comment:
```
In hack/cherry_pick_pull.sh line 36:
MAIN_REPO_NAME=${MAIN_REPO_NAME:-$(git remote get-url $UPSTREAM_REMOTE | awk -F'[@:./]' 'NR==1{print $5}')}
                                                      ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
fixed